WordPress/Classes/ViewRelated/Themes/ThemeBrowserHeaderView.swift

```diff
@@ -28,6 +30,22 @@ open class ThemeBrowserHeaderView: UICollectionReusableView {
     fileprivate var theme: Theme? {
         didSet {
             currentThemeName.text = theme?.name
+            if let theme = theme, theme.custom {
```

Comment:
Could this perhaps be consolidated a little? Something like...

```
var customTheme = false
var hasDetailsURL = false

if let theme = theme {
  customTheme = theme.custom
  hasDetailsURL = theme.hasDetailsURL()
}

supportButton.isHidden = customTheme
supportIcon.isHidden = customTheme
detailsButton.isHidden = customTheme && hasDetails
detailsIcon.isHidden = customTheme && hasDetails
```

Or you could use `let customTheme = theme?.custom ?? false` to initialize the vars... I'm not sure if this is any clearer, but it means we don't have branching logic and only assign each `isHidden` in one place. Your call!